main.go

```diff
@@ -168,6 +174,11 @@ func visit(url *url.URL) {
 		fmt.Println(grayscale(14)(k+":"), color.CyanString(strings.Join(v, ",")))
 	}
 
+	// if only header is requested, then skip other output
```

Comment:
i don't want you to return at this point, it should continue to read the response but _not_ count the time read the response. That is, -I sets the method to `HEAD` and expects there to be no body.